schema.sql

```diff
@@ -65,16 +65,21 @@ CREATE TABLE availability (
   FOREIGN KEY(listings_id) REFERENCES listings(id)
 );
 
-INSERT INTO destinations (destination_name) VALUES ("San Francisco");
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Glen Park", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Golden Gate Park", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Haight Ashbury", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Inner Richmond", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Inner Sunset", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Outer Richmond", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Outer Sunset", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Presidio", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Financial District", 1);
+INSERT INTO destinations (destination_name) VALUES ('San Francisco');
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ('Glen Park', 1);
```

Comment:
Suggestion: There is an opportunity here to reduce the repetition with a loop.

Reply:
Will leave this for now until I find a way to create loops in .sql files